src/platform/packages/shared/response-ops/rule_form/src/rule_details/rule_investigation_guide_editor.tsx

```diff
@@ -60,10 +50,19 @@ export function InvestigationGuideEditor({ setRuleParams, value }: Props) {
       value={value}
       onChange={(blob) => setRuleParams({ investigation_guide: { blob } })}
       onParse={onParse}
-      errors={errorMessages}
+      errors={isTooLong ? [CONTENT_TOO_LONG_ERROR_MESSAGE] : []}
       height={200}
       data-test-subj="investigationGuideEditor"
       initialViewMode="editing"
     />
   );
 }
+
+const CONTENT_TOO_LONG_ERROR_MESSAGE = i18n.translate(
+  'responseOpsRuleForm.investigationGuide.editor.errorMessage',
+  {
+    defaultMessage:
+      'The Investigation Guide is too long. Please shorten it.\nCurrent length: {length}.\nMax length: {maxLength}.',
```

Comment:
```suggestion
      'The investigation guide is too long. Please shorten it.\nCurrent length: {length}.\nMax length: {maxLength}.',
```
small nit on case, I know it's not new, but it doesn't need to be capitalized